tools/doc/html.mjs

```diff
@@ -34,6 +41,13 @@ import { visit } from 'unist-util-visit';
 import * as common from './common.mjs';
 import * as typeParser from './type-parser.mjs';
 
+highlightJs.registerLanguage('js', javascript);
+highlightJs.registerLanguage('c', c);
+highlightJs.registerLanguage('coffee', coffeescript);
+highlightJs.registerLanguage('cpp', cpp);
+highlightJs.registerLanguage('http', http);
+highlightJs.registerLanguage('json', json);
```

Comment:
We appear to allow more languages in code fences through the linter in ouir preset: https://github.com/nodejs/remark-preset-lint-node/blob/82217ae7822fbb06ecd2fbd56a8ed7c0bd7863ac/index.js#L50-L64

Reply:
That's correct. Some of them (e.g., `r`) don't appear in the HTML-ified docs and therefore aren't relevant here. Others (`bash` and `console` in particular) are things for which I'm removing highlighting because they do weird things in some of our docs (which is probably because we're not quite using them as they are intended).